### ISSUE TITLE
can not add map style

### ISSUE THREAD

**Issue by hizuozhen** [keplergl/kepler.gl#2458]
Base map
click add map style
Paste my own [published] style url 
msg:
Failed to load map style, make sure it is published. For private style, paste in your access token

![Dingtalk_20231130013605](https://github.com/keplergl/kepler.gl/assets/152428152/bd09dccb-3625-4e2c-b3a2-9e0de5bb8529)

**Comment by vipermax** [comment]
Similar here.
But different is I can add and preview the style, but the "Add Style" btn is not click-able.
<img width="959" alt="image" src="https://github.com/keplergl/kepler.gl/assets/57872/7e28efa1-09ee-4ecb-9531-6c9719f2bb09">

**Comment by FlorentLee** [comment]
Hi guys, I met the same bug all the time when I trying to add a style with a url of JSON/CSV. Can you please fix it as soon as possible? I really appreciate your help!

**Comment by FlorentLee** [comment]
Hi guys, I met the same bug all the time when I was trying to add a style with a url of JSON/CSV. Can you please fix it as soon as possible? I really appreciate your help!

**Comment by bkowshik** [comment]
Experienced this issue while trying to add Mapbox Satellite imagery as a Custom Map Style. 😞

**Comment by juanmaz12** [comment]
As many others, I faced this same issue. Given that Kepler moved from mapbox to maplibre, I solved my needs  using maptiler.com which uses maplibre and it's free (has Satellite map which is what I needed).

**Comment by bkowshik** [comment]
Thank you @juanmaz12 The `Add Style` button is in inactive state even after filling in the URL and name.


<img width="948" alt="image" src="https://github.com/keplergl/kepler.gl/assets/2899501/ca8b4311-7ae5-4661-91e2-66a15d3ec0fb">

**Comment by juanmaz12** [comment]
You are correct @bkowshik, tried today and Add Style is inactive. Something mush have changed since I tried with maptiler.
I had to download an older version of kepler.gl that still has mapbox support and run it locally to use it today.

**Comment by bkowshik** [comment]
@juanmaz12 Can you share the older version number of Kepler.GL that worked for you?

**Comment by juanmaz12** [comment]
Sure, I'm using release #v3.0.0-alpha.1

**Comment by chrisgervang** [comment]
Just tested my MapTiler Satellite url, and it appears to be fixed -- tested on v3.1.8
